pxr/usd/usd/crateData.cpp

```diff
@@ -803,14 +803,15 @@ class Usd_CrateDataImpl
                 liveFieldSets[FieldSetIndex(fsBegin-fieldSets.begin())];
                     
             dispatcher.Run(
-                [this, fsBegin, fsEnd, &fields, &fieldValuePairs]() mutable {
+                [this, fsBegin, fsEnd, &fields, &fieldValuePairs] {
                     // XXX Won't need first two tags when bug #132031 is
                     // addressed
                     TfAutoMallocTag tag(
                         "Usd", "Usd_CrateDataImpl::Open", "field data");
                     auto &pairs = fieldValuePairs.GetMutable();
                     pairs.resize(fsEnd-fsBegin);
-                    for (size_t i = 0; fsBegin != fsEnd; ++fsBegin, ++i) {
+                    auto fsIt = fsBegin;
+                    for (size_t i = 0; fsIt != fsEnd; ++fsIt, ++i) {
                         auto const &field = fields[fsBegin->value];
```

Comment:
now `fsBegin` is not incrementing, but still used in cycle's body - looks like the behavior will change